cmd/waku/server/rest/lightpush_rest.go

```diff
@@ -33,6 +33,11 @@ func (msg lightpushRequest) Check() error {
 	if msg.Message == nil {
 		return errors.New("waku message is required")
 	}
+
+	if msg.PubSubTopic == "" {
```

Comment:
I don't think we should have this check at REST API because in case of autosharding the pubsubTopic is filled automatically by the underlying lightpush client API. Ref code https://github.com/waku-org/go-waku/blob/3f69fb377656302855d95b278bff123f6eb17f4f/waku/v2/protocol/lightpush/waku_lightpush.go#L268-L273